### PR TITLE
Fix pylint pre commit

### PR DESCRIPTION
**Pull request by bentsi**
## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Review by vponomaryov — APPROVED** [review]

File: sdcm/cluster.py [hunk 1]
```diff
@@ -2102,14 +2103,19 @@ def install_scylla_debuginfo(self):
...
         elif self.distro.is_ubuntu or self.distro.is_debian:
             result = self.remoter.run(f'dpkg-query --show {self.scylla_pkg()}', verbose=False, ignore_status=True)
         else:
             raise ValueError(f"Unsupported Linux distribution: {self.distro}")
-        return result.exit_status == 0
+        if result.exit_status == 0:
+            return True
+        elif raise_if_not_installed:
+            raise Exception(f"There is no pre-installed ScyllaDB on {self}")
+        else:
```
Comment (dkropachev) [inline sdcm/cluster.py:2117]: Redundant else

File: sdcm/cluster.py [hunk 2]
```diff
@@ -2889,21 +2894,21 @@ def wait_for_machine_image_configured(self):
         wait.wait_for(self.is_machine_image_configured, step=10, timeout=300)
 
     def get_sysctl_output(self) -> Dict[str, str]:
```
Comment (dkropachev) [inline sdcm/cluster.py:2896]: Maybe rename it to get_sysctl_properties

**Review by dkropachev — CHANGES_REQUESTED** [review]

File: sdcm/cluster.py (hunk 1 above)
Comment (bentsi) [reply sdcm/cluster.py:2117]: fixed

File: sdcm/cluster.py (hunk 2 above)
Comment (bentsi) [reply sdcm/cluster.py]: done

**Comment by bentsi** [comment]
checked on https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/bentsi-staging/job/bentsi-longevity-3h/28/console

**Review by dkropachev — APPROVED** [review]

**Pull request merged**